src/api/models/user.js

```diff
@@ -60,6 +60,11 @@ export default (sequelize, DataTypes) => {
       onDelete: 'CASCADE',
       hooks: true
     });
+    User.hasMany(models.LikeDislike, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE',
+      hooks: true
```

Comment:
@murediane what does ``hooks: true`` do?

Reply:
Hooks are functions that are called before and after (bulk-) creation/updating/deletion and validation. 
In this case, I am simply saying here that if there hooks they must be executed. Does that make sense to you